webroot/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1>Home</h1>
+<ul>
+    <li><a href="a_web_page.html">a_web_page.html</a></li>
+    <li><a href="sample.txt">sample.txt</a></li>
+    <li><a href="make_time.py">make_time.py</a></li>
+    <li><a href="images.html">images</a></li>
+</ul>
+</body>
+</html>
```

Comment:
You hand-wrote this.  I wanted you to create it programmatically.  Now if you add a new file, you have to update this page to reflect that addition.  Let your program do that work.